lib/collection/src/operations/shared_storage_config.rs

```diff
@@ -46,7 +47,7 @@ impl Default for SharedStorageConfig {
             incoming_shard_transfers_limit: DEFAULT_IO_SHARD_TRANSFER_LIMIT,
             outgoing_shard_transfers_limit: DEFAULT_IO_SHARD_TRANSFER_LIMIT,
             snapshots_path: DEFAULT_SNAPSHOTS_PATH.to_string(),
-            s3_config: None,
+            snapshots_config: default::Default::default(),
```

Comment:
Changed from `s3_config` to `snapshots_config` so that we can extend to other storage service